### PR TITLE
School Library: Add classes

### PR DESCRIPTION
**Pull request by amare1990**
- [ ] Setup Gitflow.
- [ ] Added and setup linters.
- [ ] Added Person class 
- [ ] Added Student class that inherits from Person class.
- [ ] Added Teacher class that inherits from Person class.
- [ ] Implemented the instance variables and other methods per the project requirement.
- [ ] Updated the README.md template.
- [ ] Checked and fixed linters errors"

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, parent_permission, name, classroom)
```
Comment (prolajumokeoni) [inline student.rb]: - [ ]  The name' and 'parent_permission'  should have default values of "Unknown" and 'true', please add it
```suggestion
  def initialize(`age, classroom, name = "Unknown", parent_permission = true`)
```

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, parent_permission, name, specialization)
```
Comment (prolajumokeoni) [inline teacher.rb]: - [ ]  The name' and 'parent_permission'  should have default values of "Unknown" and 'true', please add it

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @amare1990 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good Readme
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (amare1990) [reply student.rb]: Hi @prolajumokeoni . Thanks for your suggestion.

**Review by Davidosky007 — APPROVED** [review]
Hi @amare1990 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Good looking readme file 👍 
* Proper use of Gitflow 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**